src/web/mjs/connectors/ReaperScans.mjs

```diff
@@ -6,10 +6,34 @@ export default class ReaperScans extends WordPressMadara {
         super();
         super.id = 'reaperscans';
         super.label = 'Reaper Scans';
-        this.tags = [ 'webtoon', 'english' ];
+        this.tags = ['webtoon', 'english'];
         this.url = 'https://reaperscans.com';
         this.links = {
             login: 'https://reaperscans.com/login'
         };
+        this.queryChapters = 'li.wp-manga-chapter a';
     }
+
+    async _getChapters(manga) {
```

Comment:
May consider to add a bloat removal to `WordPRessMadara` as done in `WordPressMangastream`
e.g. ronny1982@29e3e8a
And then simply define the bloat removal query in `ReaperScans`
e.g. ronny1982@d8468ee